src/Input.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+import CommonButton from './CommonButton';
+
+export default function Input({ handleClick }) {
+  const [value, setValue] = useState('');
+
+  const checkInputValue = (e) => {
+    const inputText = e.target.value;
+    return setValue(inputText);
+  };
+
+  return (
+    <div>
+      <label htmlFor="todoInput">
+        <input id="todoInput" type="text" onChange={(e) => checkInputValue(e)} />
+        <CommonButton lable="추가" handleClick={() => handleClick(value)} disabled={!value && true} />
+      </label>
```

Comment:
htmlFor 필드를 사용하신건 잘하신거같아요!! 

input 을 label 의 children 으로 두는것은 어색한거 같습니다 :)
```suggestion
      <label htmlFor="todoInput">Count</label>
      <input id="todoInput" type="text" onChange={(e) => checkInputValue(e)} />
      <CommonButton lable="추가" handleClick={() => handleClick(value)} disabled={!value && true} />
```

Reply:
- 시안에서는 label이 없기 때문에 스크린 리더에서만 읽히도록 label 태그에 style를 넣었습니다. 
- 공식문서에서는 for/id로 연결하지 않는 경우 label 태그 내부에 input 태그를 넣는 것도 하나의 예로 보여줍니다 혹 어색하다고 말씀하신 건 어떤 이유인가요? 
Associating labels implicitly(https://www.w3.org/WAI/tutorials/forms/labels/) 
 (* buttom 테그를 넣는 것은 잘못 된 것 같습니다!)

Reply:
저도 몰랐던 사실이였네요 !! 한나님 덕분에 좋은 사실을 한개 배우고 갑니다 👍